mix.exs

```diff
@@ -48,6 +49,7 @@ defmodule Hexpm.MixProject do
       {:ex_aws, "~> 2.0"},
       {:ex_machina, "~> 2.0"},
       {:eqrcode, "~> 0.1.6"},
+      {:floki, ">= 0.30.0", only: :test},
```

Comment:
What is floki used for?

Reply:
It was meant to be used for HTML parsing, but I guess it is not needed in the scope of this PR for now

Reply:
Removed.